src/components/card/Card.scss

```diff
@@ -27,13 +48,26 @@
         }
     }
     .text-block {
-        padding: 20px;
+        width: 80%;
+        padding: 20px 10px;
         color: white;
         display: flex;
         flex-direction: column;
-        justify-content: space-between;
-        p {
-            margin-top: 0;
+        justify-content: center;
+        align-items: center;
+        text-align: center;
+
+        h1 {
```

Comment:
Here you should add a property to handle word breaking if the text is too long, and handle overflow at least on the description part :)